hknweb/admin.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib import admin
+from django.contrib.auth.models import User, Group
+from django.contrib.auth.admin import UserAdmin
+from django.conf import settings
+
+# Unregister the provided model admin
+admin.site.unregister(User)
+
+# Register out own model admin, based on the default UserAdmin
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    list_display = ('username', 'first_name', 'last_name', 'email', 'officer', 'candidate')
+    readonly_fields = ('officer', 'candidate')
+
+    actions = ['add_cand', 'add_officer', 'remove_cand', 'remove_officer']
+
+    def officer(self, user):
+        return 'Y' if user.groups.filter(name=settings.OFFICER_GROUP).exists() else ''
+
+    def candidate(self, user):
+        return 'Y' if user.groups.filter(name=settings.CAND_GROUP).exists() else ''
+
+    def add_cand(self, request, queryset):
+        group = Group.objects.get(name=settings.CAND_GROUP)
+        for u in queryset:
+            group.user_set.add(u)
```

Comment:
Consider using
```py
group.user_set.update(queryset)
```
to add all users at once to the group, rather than adding them one at a time.

Reply:
How do you know all these things

Reply:
I figured there would be some function to handle iterables instead of just single elements, the way lists have .append() and .extend(), and the rest was searching the Django queryset docs.

Reply:
Uh actually this is more complicated so I'll just keep it